pkg/quarterdeck/api/v1/api.go

```diff
@@ -184,12 +184,14 @@ type VerifyRequest struct {
 //===========================================================================
 
 type Organization struct {
-	ID       ulid.ULID `json:"id"`
-	Name     string    `json:"name"`
-	Domain   string    `json:"domain"`
-	Projects int       `json:"projects"`
-	Created  time.Time `json:"created,omitempty"`
-	Modified time.Time `json:"modified,omitempty"`
+	ID        ulid.ULID `json:"id"`
+	Name      string    `json:"name"`
+	Domain    string    `json:"domain"`
+	Owner     string    `json:"owner"`
```

Comment:
Can't an organization have many owners?

Reply:
Yes, unless we want this to be a `created_by`. But I removed the field for now since we don't have a way of representing that in the database yet.